modules/server_new/src/main/scala/observe/server/tcs/package.scala

```diff
@@ -1,78 +1,100 @@
-package observe.server
+package observe.server.tcs
 
+import algebra.instances.all.given
 import cats.Eq
 import cats.syntax.all.*
+import coulomb.*
+import coulomb.policy.standard.given
+import coulomb.syntax.*
+import coulomb.units.accepted.ArcSecond
+import coulomb.units.accepted.Millimeter
 import edu.gemini.observe.server.tcs.BinaryEnabledDisabled
 import edu.gemini.observe.server.tcs.BinaryOnOff
 import edu.gemini.observe.server.tcs.BinaryYesNo
 import edu.gemini.observe.server.tcs.ParkState
+import lucuma.core.math.Angle
+import lucuma.core.math.Offset
 import lucuma.core.util.TimeSpan
-import squants.Angle
-import squants.Length
-import squants.Ratio
-import squants.space.Arcseconds
-import squants.space.Millimeters
+import observe.server.tcs.TcsController.InstrumentOffset
+import observe.server.tcs.TcsController.OffsetP
+import observe.server.tcs.TcsController.OffsetQ
 
 import java.time.temporal.ChronoUnit
 
-package tcs {
+private class FocalPlaneScale(
+  angle:  Quantity[Double, ArcSecond],
+  length: Quantity[Double, Millimeter]
+) {
 
-  final class FocalPlaneScale(angle: Angle, length: Length) extends Ratio[Angle, Length] {
+  def base: Quantity[Double, ArcSecond] = angle
 
-    override def base: Angle = angle
+  def counter: Quantity[Double, Millimeter] = length
 
-    override def counter: Length = length
+  def times(l: Quantity[Double, Millimeter]): Quantity[Double, ArcSecond] =
+    (l / length) * angle
 
-    def times(l: Length): Angle = convertToBase(l)
-    def *(l:     Length): Angle = times(l)
+  def *(l: Quantity[Double, Millimeter]): Quantity[Double, ArcSecond] = times(l)
 
-    def divide(a: Angle): Length = convertToCounter(a)
-  }
+  def divide(a: Quantity[Double, ArcSecond]): Quantity[Double, Millimeter] =
+    counter * (a / base)
 
-  object FocalPlaneScale {
+}
 
-    implicit class AngleOps(a: Angle) {
-      def dividedBy(fps: FocalPlaneScale): Length = fps.divide(a)
-      def /(fps:         FocalPlaneScale): Length = dividedBy(fps)
-    }
+val tcsTimeout: TimeSpan = TimeSpan.unsafeFromDuration(90, ChronoUnit.SECONDS)
+val agTimeout: TimeSpan  = TimeSpan.unsafeFromDuration(90, ChronoUnit.SECONDS)
 
-    implicit class LengthOps(l: Length) {
-      def times(fps: FocalPlaneScale): Angle = fps * l
-      def *(fps:     FocalPlaneScale): Angle = times(fps)
-    }
+object FocalPlaneScale:
+  extension (l: Quantity[Double, Millimeter])
+    def times(fps: FocalPlaneScale): Quantity[Double, ArcSecond] = fps * l
+    def **(fps:    FocalPlaneScale): Quantity[Double, ArcSecond] = times(fps)
```

Comment:
I'd discourage using `**` as multiplication. It can be confusing since it's exponentiation in some languages (eg: Python, JS).

Reply:
right, the issue is that Quantity has already a `*` operator and the compiler would be all confused. What about `:*`?